net/netstinky/Makefile

```diff
@@ -0,0 +1,63 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netstinky
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=nsids-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/wanduow/netstinky-ids/releases/download/v1.0.0/
+PKG_HASH:=143e8d7a9ee7f58893d92b065aed7eca35798f5807b07d9a78f404ea8614e216
+PKG_BUILD_DIR:=$(BUILD_DIR)/nsids-$(PKG_VERSION)
```

Comment:
Is this necessary?

Reply:
It appears to be. The package is named `netstinky` but the tarball is `nsids` (NetStinky IDS) when I remove the PKG_BUILD_DIR it will untar into nsids-1.0.0 but then tries to build in netstinky-1.0.0.